CHANGELOG.md

```diff
@@ -17,6 +17,7 @@ All notable changes to this project will be documented in this file.
 - Setup sms-related classes. (#15)
 - Setup result-related classes. (#16)
 - Setup compute- and tee-related classes. (#17)
+- Setup pubsub-related classes. (#18)

```

Comment:
We should find another wording and maybe replace the PR number with another one as it is no more by the PR under review.

Reply:
```suggestion
- Setup pubsub-related classes. (#25)
```